packages/datepickers/src/elements/Datepicker/Datepicker.tsx

```diff
@@ -192,45 +190,39 @@ const Datepicker: React.FunctionComponent<IDatepickerProps & ThemeProps<DefaultT
             });
           }}
         </Reference>
-        {state.isOpen && (
-          <Popper
-            placement={popperPlacement as any}
-            modifiers={popperModifiers}
-            // Disable position updating on scroll events while menu is closed
-            eventsEnabled={state.isOpen && eventsEnabled}
-          >
-            {({ ref, style, scheduleUpdate, placement: currentPlacement }) => {
-              scheduleUpdateRef.current = scheduleUpdate;
 
-              const popperStyle = { ...style, zIndex };
+        <Popper
+          placement={popperPlacement as any}
```

Comment:
I think the type cast may not be needed because `popperPlacement` is a result of functions that return `POPPER_PLACEMENT`. Would you be able to confirm if that's the case?

Reply:
I've removed this cast.